### ISSUE TITLE
Refine/align coloring of body text and inline text links

### ISSUE THREAD

**Issue by iamthomasbishop** [wordpress-mobile/gutenberg-mobile#697]
Current vs. Proposed

<img width="924" alt="image" src="https://user-images.githubusercontent.com/1200432/53759690-8cc87d00-3e86-11e9-99b0-fe3f38f0c4da.png">

## Body Text

Currently, all body-text (headings, paragraph text, image captions, etc) are set to `#000000` (black), which is a bit too strong. To align with an upcoming colors refactor in the apps, let's align with that proposal of `#1A1A1A` aka `$gray-900` – from [this palette](https://automattic.github.io/color-studio/).

## Inline Text Links

The color of inline text links is currently something around `#027AFF` (see left on example above). Let's switch to use `$blue-500` (`#016087`) for alignment. 

I've voiced a concern that this blue feels like it doesn't provide enough contrast against the `$gray-900` text, especially when not underlined, but it's an improvement, and it passes color contrast tests on a white background.

Note: we _aren't currently underlining text links on Android_ – we should change this for accessibility sake.

## Punch List

- [ ] Change body text to `#1A1A1A` aka `$gray-900` in [this palette](https://automattic.github.io/color-studio/)
- [ ] Change inline link color to `#016087` aka `$blue-500` in [this palette](https://automattic.github.io/color-studio/)
- [ ] Force underline to links in Android
